test/script.js

```diff
@@ -1,6 +1,5 @@
 // test/script.js
 
-const teststring = require('trimmiddle')
-
-console.log(teststring('This is a long string that will be trimmed'));
+const teststring = require("trimmiddle");
 
+console.log(teststring("This is a long string that will be trimmed"));
```

Comment:
I wasn't sure how you execute this script, so I created a local variant `script.cjs` and ran it through Node:

```javascript
const { trimMiddle } = require("../index.js")

console.log(trimMiddle("This is a long string that will be trimmed"));
```

(Note that the import differs.)

Might be worth documenting the process in `README.md`?

Reply:
test suite is a thing to work on, already there was a report of an edge case bug.